.github/ISSUE_TEMPLATE/qg-checklist.md

```diff
@@ -32,6 +32,10 @@ Release Management Reference Issue: <!-- Note: Add the related product RM issue
 - [ ] [TRG 1.02](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-2) appropriate install instructions either `INSTALL.md` or in `README.md`
 - [ ] [TRG 1.03](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-3) appropriate `CHANGELOG.md`
 - [ ] [TRG 1.04](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-4) editable static files
+- [ ] [TRG 1.05](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-5) architecture docs
```

Comment:
new links are wrong. Those are `05`, `06`, `07` and `08`. That's likely the reason for the wrong ordering on github page, too.

Reply:
ahhh

Reply:
but now what shall we fix the original links in the page or here the link? what makes more sense?

Reply:
align within TRG to without `0` so that the ordering also works properly, I'd say

Reply:
Yes, but if we already need to update something, why not making it fully consistent? The extra effort is low

Reply:
With fully consistent you mean for all TRGs either with or without `0`?

Reply:
shall I change the links now and we discuss this later?

Reply:
Fine for me to discuss this later
Also fine all TRGs with leading `0` --> maybe change after release 24.08?

Reply:
Yeah, let's make it without hurry (Y)

Reply:
fixed